rest/routes/points_sum.py

```diff
@@ -0,0 +1,59 @@
+"""This module contains endpoint for getting recruitment points of candidates"""
+import logging
+from flask import Blueprint, request
+from rest.common.token import handle_request_token
+from rest.common.response import create_response
+from rest.models.candidate_recruitment import CandidateRecruitment
+from rest.models.recruitment import Recruitment
+
+
+points_sum_url = Blueprint('points', __name__)
+
+
+@points_sum_url.route('', methods=['POST', 'OPTIONS'])
```

Comment:
POST method shouldn't be used for fetching data. Use GET.

Reply:
Did not notice that, @twistzzmc please fix this